src/routes/user.js

```diff
@@ -1,11 +1,15 @@
 import express from 'express';
-import userValidator from '../middlewares/userValidator';
-import signUp from '../controllers/userController';
+import middlewares from '../middlewares';
+import userController from '../controllers/userController';
 
+const { userValidator: { signUpValidator } } = middlewares;
+const { signUp, profile } = userController;
 const user = express.Router();
 
-const middlewares = [userValidator.signUp];
+// Route to sign up a user
+user.post('/users', signUpValidator, signUp);
 
-user.post('/users', middlewares, signUp);
+// Route to get user profiles by username
+user.get('/profiles/:username', profile);
```

Comment:
I noticed there is no validation middleware on this route to ensure that the username entered isn't empty and is valid.

Reply:
Noted, would work on it

Reply:
Done